tests/std/run.pl

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+use strict;
+use File::Basename;
+
+# This file is just a stub to find and execute the real run.pl which is

```

Comment:
This suggests we forgot a file.

Reply:
It refers to a file on the Microsoft-internal side of things; I can clean up this comment though (as I believe it provides no value).

Reply:
Can we copy the real run.pl and run.pm here?

Reply:
That would be significant Perl surgery which I would prefer not to do. We virtually never need to modify those files when updating tests - the only times I've had to are when we've added new compilers like Clang. Ultimately, the new test runner that @cbezault is working on will be used by contributors to this GitHub repo, and the Perl infrastructure will be a minor curiosity (especially if they're driven by the same `env.lst` files, which are the main thing we need to modify when adding tests). Hopefully we can make the Microsoft-internal test runner Contest use the GitHub test runner and eliminate this Perl dependency (much as we hope to make the Microsoft-internal MSBuild system use the CMake build system and eliminate `stl/msbuild` which is a similar relic).

I considered filing an issue to track this, but I don't think we should perform such surgery at all.